scripts/proto-copy-cosmos-sdk.sh

```diff
@@ -13,6 +13,11 @@ find "$ibc_protos" -maxdepth 1 -mindepth 1 \
   -exec sh -c 'file=$1; basename $1  | xargs -I % -n1 rm -rf "./third_party/proto/%"' _ {} \;
 cp -r "$ibc_protos" ./third_party
 
+wasm_protos="$(go list -f '{{ .Dir }}' -m github.com/CosmWasm/wasmd)/proto/cosmwasm"
+find "$wasm_protos" -maxdepth 1 -mindepth 1 \
+  -exec sh -c 'file=$1; basename $1  | xargs -I % -n1 rm -rf "./third_party/proto/%"' _ {} \;
+cp -r "$wasm_protos" ./third_party/proto
```

Comment:
is `third_party/proto` correct? Cosmos copies directly into `third_party`

Reply:
I'm pretty sure this is going to break our proto root

Reply:
yeah, proto is a subfolder
https://github.com/axelarnetwork/axelar-core/tree/main/third_party/proto

github skips directly to `proto` subfolder because there's no other folders in `third_party`

Reply:
oh nvm, i misunderstood, proto folder has some extra files that I didn't want to pull in
https://github.com/CosmWasm/wasmd/tree/main/proto
So, I'm populating it directly in the proto subfolder